develop/TOOLS/DIS/DIS_menu.md

```diff
@@ -30,7 +30,6 @@ The following sections provide more information on the DIS menu:
 - [Tool Windows \> DIS Validator](xref:DisValidatorToolWindow)
 - [Tool Windows \> DIS Comparer](xref:DisComparerToolWindow)
 - [Settings](xref:DIS_settings)
-- [Check for updates...](#check-for-updates)
```

Comment:
As the 'Generate Class Library code' stuff was also kept (with the message in which version it was removed), maybe keep it as well? Or let's remove the class library stuff as well?